src/page/Main.tsx

```diff
@@ -10,14 +10,18 @@ import Home from "./Home";
 import Patient from "./Patient";
 import Variant from "./Variant";
 import Api from "./Api";
+import { VariantStore } from './VariantStore';
+import { observer } from 'mobx-react';
 
+@observer
 class Main extends React.Component<{}>
 {
     public render()
     {
         const VariantPage = (props: any) => (
             <Variant
                 variant={props.match.params.variant}
+                store={new VariantStore(props.match.params.variant)}
```

Comment:
might be better to add a separate getter for this:
```suggestion
        @computed
        get variantStore() {
            return new VariantStore(props.match.params.variant);
        }
```
```suggestion
                store={this.variantStore}
```

Reply:
@onursumer I need to put the `get variantStore()` outside the render right? how can I get the `props.match.params.variant` then?